src/lib/utils/api/zeroex.ts

```diff
@@ -7,9 +7,13 @@ import {
   ZeroExAffiliateAddress,
 } from '@/constants/server'
 
-export function getConfiguredZeroExApi(swapPathOverride: string): ZeroExApi {
+export function getConfiguredZeroExSwapQuoteProvider(
+  chainId: number,
+): ZeroExSwapQuoteProvider {
+  const networkKey = getNetworkKey(chainId)
+  const swapPathOverride = `/${networkKey}/swap/v1/quote`
   const headers = getIndexApiHeaders()
-  return new ZeroExApi(
+  return new ZeroExSwapQuoteProvider(
```

Comment:
ZeroExSwapQuoteProvider is configured just as `ZeroExApi` before.

_In the future that's where we'd plug-in a LiFiSwapQuoteProvider once it's created._